include/industry/foc/fixed16/foc_feedforward.h

```diff
@@ -33,6 +33,11 @@
  * Public Type Definition
  ****************************************************************************/
 
+#ifdef CONFIG_DEBUG_UORB
```

Comment:
why change this file

Reply:
Sorry, no need, a mistake while processing merge conflicts / depends.